install.ps1

```diff
@@ -23,7 +24,8 @@ Install-ChocoPackage python2
 Install-ChocoPackage nodejs-lts
 
 # Installing Git
-choco install -y git
+Install-ChocoPackage git
+Update-SessionEnvironment
  $env:PATH = 'C:\Program Files\Git\cmd;{0}' -f $env:PATH ;
   [Environment]::SetEnvironmentVariable('PATH', $env:PATH, [EnvironmentVariableTarget]::Machine)
```

Comment:
These two lines (29, 30) should be removed. It's no longer necessary after update-sessionenvironment is run.

Reply:
ok